cookbook/core/type_system/structured_dataset.py

```diff
@@ -87,18 +87,80 @@ def get_subset_df(
 ) -> Annotated[StructuredDataset, subset_cols]:
     df = df.open(pd.DataFrame).all()
     df = pd.concat([df, pd.DataFrame([[30]], columns=["Age"])])
-    # On specifying BigQuery uri for StructuredDataset, Flytekit will write pd.dataframe to a BigQuery table
+    # On specifying BigQuery uri for StructuredDataset, flytekit writes pd.dataframe to a BigQuery table
     return StructuredDataset(dataframe=df)
 
 
 # %%
+# StructuredDataset with ``uri`` Argument
+# ========================================
+#
+# If you specify BigQuery ``uri`` for StructuredDataset, flytekit writes the pandas dataframe to a BigQuery table.
+#
+# Before writing DataFrame to a BigQuery table,
+#
+# #. Create a `GCP account <https://cloud.google.com/docs/authentication/getting-started>`__ and create a service account.
+# #. Create a project and add the ``GOOGLE_APPLICATION_CREDENTIALS`` environment variable to your bashrc file.
+# #. Create a dataset in your project.
+#
+# Let's understand how to create a DataFrame from a BigQuery table through an example.
```

Comment:
Please remove this as the introduction can be specified above the bullet points.